mars/dataframe/expressions/arithmetic/core.py

```diff
@@ -585,3 +587,226 @@ def __call__(self, x1, x2):
 
     def rcall(self, x1, x2):
         return self._call(x1, x2)
+
+class DataFrameUnaryOpMixin(DataFrameOperandMixin):
```

Comment:
https://github.com/mars-project/mars/blob/c87d4819e9712c70d1b6ff5e98f2e4b1cadfacaa/mars/tensor/expressions/arithmetic/core.py#L296-L298

I think we can add `DataFrameBinOp` and `DataFrameUnaryOp` with the lhs and rhs attribute.